github/authorizations.go

```diff
@@ -121,7 +121,7 @@ func (a AuthorizationRequest) String() string {
 // fields. That is, you may provide only one of "Scopes", or "AddScopes", or
 // "RemoveScopes".
 //
-// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/oauth_authorizations/#update-an-existing-authorization
+// GitHub API docs: https://docs.github.com/en/rest/oauth_authorizations/#update-an-existing-authorization
```

Comment:
This moved to https://docs.github.com/en/rest/oauth-authorizations#update-an-existing-authorization